src/preferences/dialog/dlgprefinterface.cpp

```diff
@@ -25,12 +25,14 @@ namespace {
 
 const QString kConfigGroup = QStringLiteral("[Config]");
 const QString kControlsGroup = QStringLiteral("[Controls]");
+const QString kPreferencesGroup = QStringLiteral("[Preferences]");
 const QString kScaleFactorKey = QStringLiteral("ScaleFactor");
 const QString kStartInFullscreenKey = QStringLiteral("StartInFullscreen");
 const QString kSchemeKey = QStringLiteral("Scheme");
 const QString kResizableSkinKey = QStringLiteral("ResizableSkin");
 const QString kLocaleKey = QStringLiteral("Locale");
 const QString kTooltipsKey = QStringLiteral("Tooltips");
+const QString kMultiSamplingKey = QStringLiteral("multi_sampling");
```

Comment:
consistency...
```suggestion
const QString kMultiSamplingKey = QStringLiteral("MultiSampling");
```

Reply:
That's what I started with, and then I realized than other keys in `[Preference]` were using a lower snake case formatting. Is the capitalized pascal case now enforce for every new setting keys?

Reply:
I think the name could be improved, this could also be a boolean value. How about something like `MultiSamplingFactor`?

Reply:
Sounds good to me. 
Do you concur that capitalized pascal case is now the new naming standard for new settings keys?

Reply:
Sorry, I forgot to respond earlier. I only looked at the context of the file in regards to the naming scheme and assumed the rest was consistent. Lets not make this a discussion and keep it as nobody will look at this anyways.